### PR TITLE
Feature/article model

### PR DESCRIPTION
**Pull request by anusharanganathan**
This PR is for https://github.com/CottageLabs/willow/issues/138

### CODE REVIEW

File: willow/app/inputs/rights_statement_input.rb [hunk 1]
```diff
@@ -21,35 +21,35 @@ def build_components(attribute_name, value, index, options)
...
+      field_name = name_for(attribute_name, index, field)
+      field_value = rights_statement.send(field).first
 
-      # out << "<div class='row'>"
-      # out << "  <div class='col-md-3'>"
-      # out << template.label_tag(field_name, field.to_s.humanize, required: false)
-      # out << '  </div>'
+      out << "<div class='row'>"
+      out << "  <div class='col-md-3'>"
+      out << template.label_tag(field_name, 'License statement', required: false)
```
Comment (martyn-w) [inline willow/app/inputs/rights_statement_input.rb]: I think it might be 'Licence' not 'License'?

Comment (martyn-w) [inline willow/app/inputs/rights_statement_input.rb]: though not sure... I see dublin core use 'license' http://dublincore.org/documents/dcmi-terms/#terms-license

Comment (anusharanganathan) [inline willow/app/inputs/rights_statement_input.rb]: American versus UK spelling. I'll go with your recommendation

**Comment by anusharanganathan** [comment]
@martyn-w I've made changes to the article model, which hopefully address all the issues you raised. I've given details to the points you raised as a response in the comment.

**Comment by martyn-w** [comment]
Hi @anusharanganathan,

thanks for making these changes - I'm afraid I still get an error though if I create a new article but don't set the licence start date (which is not a mandatory field):

screen shot 2017-07-02 at 17 03 57

**Comment by martyn-w** [comment]
Also to add, if the licence statement is left blank, the follow error results:

screen shot 2017-07-02 at 17 20 17

**Comment by anusharanganathan** [comment]
Hi @martyn-w, I am really sorry. That was sloppy of me. I do hope I've done a better job testing my code this time. 

The changes I've made to address the issues you were seeing are in commits https://github.com/CottageLabs/willow/pull/180/commits/ec55fe088a1b1db0702b31eec443c82f110759df and https://github.com/CottageLabs/willow/pull/180/commits/fe9f0ac34df48379b21aa7d6f16cc77f569658f7. Is there a better way to do the check? It doesn't look pretty.

I also fixed another issue with the edit form - Required check to be done only for first entry for the field - commit https://github.com/CottageLabs/willow/pull/180/commits/2c240bb3af8650d4a3059255a33867588a15972f. The add another and delete will follow in a PR after this is out of the way

I moved labels and field hints to he locale file - commit https://github.com/CottageLabs/willow/pull/180/commits/e4cc3d518a54c7c1fe199bc836f1a48a090d222a

**Comment by martyn-w** [comment]
Hi @anusharanganathan, this is tested and working, excellent job 👍

**Pull request merged**